typescript/lib/modules/lambda_context.ts

```diff
@@ -0,0 +1,41 @@
+import { ApiContainerServiceClient } from "../..//kurtosis_core_rpc_api_bindings/api_container_service_grpc_pb";
+import { ExecuteLambdaArgs, ExecuteLambdaResponse } from "../../kurtosis_core_rpc_api_bindings/api_container_service_pb";
+import { newExecuteLambdaArgs } from "../constructor_calls";
+import { okAsync, errAsync, ResultAsync, ok, err, Result } from "neverthrow";
+import * as grpc from "grpc";
+
+export type LambdaID = string;
+
+// Docs available at https://docs.kurtosistech.com/kurtosis-libs/lib-documentation
+export class LambdaContext {
+	private readonly client: ApiContainerServiceClient;
+	private readonly lambdaId: LambdaID;
+
+    constructor (client: ApiContainerServiceClient, lambdaId: LambdaID) {
+        this.client = client;
+        this.lambdaId = lambdaId;
+    }
+
+    // Docs available at https://docs.kurtosistech.com/kurtosis-libs/lib-documentation
+    public async execute(serializedParams: string): Promise<Result<string, Error>> {
+        const args: ExecuteLambdaArgs = newExecuteLambdaArgs(this.lambdaId, serializedParams);
+
+        const promiseExecuteLambda: Promise<ResultAsync<ExecuteLambdaResponse, Error>> = new Promise((resolve, _unusedReject) => {
+            this.client.executeLambda(args, (error: grpc.ServiceError, response: ExecuteLambdaResponse) => {
+                if (error) {
```

Comment:
Question for this, and every other callback - this `if (error)` relies on an implicit "truthy-ness" ( https://developer.mozilla.org/en-US/docs/Glossary/Truthy ). This is very not-explicit. Instead, can we figure out what the generated bindings consider a success and be explicit about that? This will probably involve inverting the `if` E.g.:
```
if (error === null) {
    resolve(okAsync(response))
} else {
    resolve(errAsync(error))
}
```
The benefit of this is that it's both explicit and a tight bound - in EXACTLY the one case that the ts-protoc-gen documentation defines will we return an `ok`, and in all other cases we'll return an `err`. This follows the principle of "fail loudly"